src/openfl/_internal/stage3D/opengl/GLContext3D.hx

```diff
@@ -242,8 +242,10 @@ class GLContext3D {
 		
 		__updateBackbufferViewport ();
 		
-		context.backBufferWidth = width;
-		context.backBufferHeight = height;
+		var scale = Std.int (context.__stage3D.__stage.window.scale);
```

Comment:
What if the `scale` is `2.5` or something? :) Shouldn't we instead do `Std.int` on `context.backBufferWidth/Height` assignments?

Reply:
Don't know if that's possible, but anyways update the PR